ros/src/util/packages/runtime_manager/scripts/runtime_manager_dialog.py

```diff
@@ -1912,6 +1912,15 @@ def OnApexReportSend(self, event):
                     'std_msgs/String',
                     value ])
 
+	def OnInitNDT(self, event):
+		subprocess.Popen([
+			'rostopic',
+			'pub',
+			'-1',
+			'/gnss_pose_update',
+			'std_msgs/Bool',
+			"True"])
```

Comment:
since this is a python ros node it would actually be better to use rospy to publish the message: https://github.com/CPFL/Autoware/blob/c9db2feb17b16334a7d6385ee79f54b1210c77a5/ros/src/util/packages/runtime_manager/scripts/runtime_manager_dialog.py#L136.